staff/sergio-costa/e-commerce-template/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>e-commerce</title>
+</head>
+
+<body>
+    <header>
+        <i class="fab fa-css3-alt" id="logoCSS"><span>CSS</span></i>
```

Comment:
instead of ids it's better to use classes, so you could add another class to the i tags, as  i class="fab fa-css3-alt logoCSS"

Reply:
👍